userspace/libscap/scap_machine_info.h

```diff
@@ -43,6 +43,7 @@ typedef struct _scap_machine_info
 	uint64_t memory_size_bytes; ///< Physical memory size
 	uint64_t max_pid; ///< Highest PID number on this machine
 	char hostname[128]; ///< The machine hostname
+	uint64_t hz; ///< Clock ticks per second on this machine
```

Comment:
So sorry, we can't add new items without removing a reserved one because of the scap files.